core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -455,6 +460,15 @@ TEE_Result stm32_get_iwdg_otp_config(paddr_t pbase,
 	otp_data->disable_on_standby = otp_value &
 				       BIT(idx + HW2_OTP_IWDG_FZ_STANDBY_SHIFT);
 
+#ifdef CFG_STM32MP15
+	/* Device in Secure Closed state require RCC secure hardening */
+	if (stm32_bsec_get_state(&state))
+		panic();
```

Comment:
why but it in a iwdg related function?

Either put it in `init_stm32mp1_drivers()` or create a new service?

Reply:
Absolutely right, i'll move it here (I have messed something here)